### ISSUE TITLE
Set insecureEdgeTerminationPolicy when termination policy is not set in CR

### ISSUE THREAD

**Issue by leochr** [OpenLiberty/open-liberty-operator#463]
insecureEdgeTerminationPolicy setting is not respected when CR is configured with:

```
manageTLS: true
route:
    host: myhost
    insecureEdgeTerminationPolicy: redirect
```

The `insecureEdgeTerminationPolicy` is only respected when `termination` value is also specified under the `route` configuration.

- [x] RCO
- [x] OLO
- [x] WLO
- [x] tests

**Comment by leochr** [comment]
@mtambol This could be the issue you mentioned in the meeting today. This is addressed in the 1.3.0 pre-release driver. FYI @danielguinan1

**Comment by leochr** [comment]
Delivered with 1.3.0. Closing.
